### PR TITLE
Support both dev and prod

### PR DESCRIPTION
**Pull request by vincentni**

### CODE REVIEW

File: pkg/curatedpackages/packagecontrollerclient.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ func (pc *PackageControllerClient) GetCuratedPackagesRegistries() (sourceRegistr
 			defaultImageRegistry = gatedOCINamespace
 		}
 	} else {
-		defaultImageRegistry = strings.ReplaceAll(constants.DefaultCuratedPackagesRegistryRegex, "*", pc.eksaRegion)
+		defaultImageRegistry = strings.ReplaceAll(defaultImageRegistry, "us-west-2", pc.eksaRegion)
```
Comment (junshun) [inline pkg/curatedpackages/packagecontrollerclient.go:148]: Is there region replication yet on the dev images?

Comment (TerryHowe) [reply pkg/curatedpackages/packagecontrollerclient.go]: @jonahjon would know, but either way we should support it and have replication to at least one region.

**Comment by junshun** [comment]
/lgtm
/approve

We can add support for dev registry mirroring later

**Pull request merged**